app/services/WhitelistSessionCache.scala

```diff
@@ -0,0 +1,51 @@
+/*
```

Comment:
It might be an idea to change the file name (as no doubt it is no longer considered politically correct).